lec15-data-viz.Rmd

```diff
@@ -1,4 +1,341 @@
+
 ---
-title: "Data visualization in R"
+title: Data visualization in R
+author: Lindsay Coome
 ---
 
+## Lesson preamble
+
+> ### Learning Objectives
+>
+> - Making better plots for outreach and communication.
+> - Review how to produce scatter plots with trendlines using ggplot.
+> - Advanced faceting and segmenting data.
+> - Altering asthetic features in ggplot.
+> - Make bar and other graphs using ggplot.
+> - Making interactive and quick graphs.
+> - Saving graphs.
+> 
+> ### Lesson outline
+>
+> - How to visualize data (20 min)
+> - ggplot basics (10 min)
+> - Basics and faceting (10 min)
+> - Aesthetics and plot types (25 min)
+> - Interactive and quick graphing (10 min)
+> - Saving graphs (5 min)
```

Comment:
Might want to include a `Setup` section on making to have `plotly` installed.

Reply:
good point, done